src/uu/tail/src/tail.rs

```diff
@@ -553,20 +566,24 @@ fn bounded_tail(file: &mut File, mode: &FilterMode, beginning: bool) {
 ///
 /// If any element of `iter` is an [`Err`], then this function panics.
 fn unbounded_tail_collect<T, E>(
-    iter: impl Iterator<Item = Result<T, E>>,
-    count: usize,
+    mut iter: impl Iterator<Item = Result<T, E>>,
+    count: u64,
     beginning: bool,
-) -> VecDeque<T>
+) -> UResult<VecDeque<T>>
 where
     E: fmt::Debug,
 {
     if beginning {
         // GNU `tail` seems to index bytes and lines starting at 1, not
         // at 0. It seems to treat `+0` and `+1` as the same thing.
         let i = count.max(1) - 1;
-        iter.skip(i as usize).map(|r| r.unwrap()).collect()
+        skip_u64(&mut iter, i);
+        Ok(iter.map(|r| r.unwrap()).collect())
     } else {
-        RingBuffer::from_iter(iter.map(|r| r.unwrap()), count as usize).data
+        let count: usize = count
+            .try_into()
+            .map_err(|_| USimpleError::new(1, "Insufficient addressable memory"))?;
```

Comment:
@sylvestre This one also seems like a false warning - there's a test checking exactly that - https://github.com/uutils/coreutils/blob/fa608983542632b7b32b143f7736fad5804c6af8/tests/by-util/test_tail.rs#L480-L490

Reply:
That test has `#[cfg(target_pointer_width = "32")]` so it probably doesn't run on the codecov CI if that is a 64 bit platform.

Reply:
Good point. Well as you're not demanding 0 coverage warnings I guess it's alright to just ignore it